frog/imports/ui/GraphEditor/fixedComponents.js

```diff
@@ -58,21 +58,29 @@ export const LevelLines = connect(({
 
 export const TimeScale = connect(({
   store: { ui: { scale }, graphDuration }
-}) => (
-  <g>
-    {[...Array(graphDuration).keys()].map(index => {
-      const i = index + 1;
-      const boolToBit = b => b ? 1 : 0;
-      const length = boolToBit(i % 15 === 0) * 15 +
-        boolToBit(i % 5 === 0) * 10 +
-        5;
-      const x = timeToPx(i, scale);
-      return (
-        <g key={i}>
-          <line x1={x} y1={600 - length} x2={x} y2={600} stroke="grey" />
-          {i % 15 === 0 ? <text x={x - 15} y={540}>{i + ' min.'}</text> : null}
-        </g>
-      );
-    })}
-  </g>
-));
+}) => {
+  let divider = Math.round(5 / scale) % 5 * 5;
+  if (divider === 0) {
+    divider = 20;
+  }
+  return (
+    <g>
+      {[...Array(graphDuration).keys()].map(index => {
+        const i = index + 1;
+        const boolToBit = b => b ? 1 : 0;
+        const length = boolToBit(i % 15 === 0) * 15 +
+          boolToBit(i % 5 === 0) * 10 +
+          5;
```

Comment:
I find that having the displayed text not matching (every `divider` minutes) with the longer ticks (every 15 minutes) is misleading.

```
const length = (i % divider ? 15 : 0) + (i % 5 ? 10 : 0) + 5
```
Also, we can remove this `boolToBit` function